components/globe-component/particles.js

```diff
@@ -61,8 +61,10 @@ class Particles {
   }
 
   load (finished) {
-    const that = this
-
+    // Load data in format:
+    // -89.772727273,-60.0,0.0,0.0,0.0,0.0,0.0,0.0
+    // -89.772727273,60.0,0.0,0.0,0.0,0.0,0.0,0.0
+    // -89.318181818,-140.0,0.0,0.0,0.0,0.0,0.0,0.0
     loadData('https://s3-eu-west-1.amazonaws.com/deltares-opendata/wwa/wri/land.csv', (err, result) => {
```

Comment:
this file is a whopping 4,5MB / 648KB Gzipped  😱 
Out of scope for this PR, but we should take a look at it nonetheless

Reply:
It's not gzipped at the moment. There's a few optimizations. It now contains:
- too many decimals
- data for all variables in 1 file
- uncompressed data

In this case I think we can encode the data in an image. I'll implement that in a separate issue/PR.

Reply:
Put in the backlog as WWA-124